message.py

```diff
@@ -23,17 +24,15 @@ def gsi(self, attname, wait=True):
         Create a GSI for attname, named AttnameIndex; wait until done if wait
         """
         idxname = attname.capitalize()
+        AttributeDefinitions = [{'AttributeName': attname, 'AttributeType': 'S'}]
```

Comment:
Is this naming style up to pep8?

Reply:
Maybe not @jdq22 -- I'm torn.  It's probably not the way Python wants it, but it is the name of the variables that the DDB boto3 library wants it.  Do we want to hew close to the library variable names or the PEP8 variable names?

Reply:
Though actually I don't see conflict between these var names and PEP8 so I think we're good either way